src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/xunit.runner.json

```diff
@@ -1,4 +1,6 @@
 ﻿{
+  "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
   "methodDisplay": "method",
+  "longRunningTestSeconds": 120,
```

Comment:
This gives us a little notification as to which test was being slow but shouldn't otherwise change behavior. Might be helpful.